roles/installer/tasks/resources_configuration.yml

```diff
@@ -27,6 +27,55 @@
   set_fact:
     _control_plane_ee_image: "{{ _custom_control_plane_ee_image | default(lookup('env', 'RELATED_IMAGE_CONTROL_PLANE_EE')) | default(_control_plane_ee_image, true) }}"
 
+- name: Check for Receptor CA Secret
+  k8s_info:
+    kind: Secret
+    namespace: '{{ ansible_operator_meta.namespace }}'
+    name: '{{ ansible_operator_meta.name }}-receptor-ca'
+  register: _receptor_ca
+  no_log: "{{ no_log }}"
+
+- name: Create Receptor Mesh CA
+  block:
+    - name: Generate Receptor CA key
+      command: |
+        openssl genrsa -out /tmp/{{ ansible_operator_meta.name }}-receptor-ca.key 4096
+    - name: Generate Receptor CA cert
+      command: |
+        openssl req -x509 -new -nodes -key /tmp/{{ ansible_operator_meta.name }}-receptor-ca.key \
+          -subj "/CN={{ ansible_operator_meta.name }} Receptor Root CA" \
+          -sha256 -days 3650 -out /tmp/{{ ansible_operator_meta.name }}-receptor-ca.crt
+    - name: Create Receptor CA secret
+      k8s:
+        apply: true
+        definition: "{{ lookup('template', 'secrets/receptor_ca_secret.yaml.j2') }}"
+      no_log: "{{ no_log }}"
+  when: not _receptor_ca['resources'] | default([]) | length
+
+- name: Check for Receptor work signing Secret
+  k8s_info:
+    kind: Secret
+    namespace: '{{ ansible_operator_meta.namespace }}'
+    name: '{{ ansible_operator_meta.name }}-receptor-work-signing'
+  register: _receptor_work_signing
+  no_log: "{{ no_log }}"
+
+- name: Generate Receptor work signing RSA keypair
+  block:
+    - name: Generate Receptor work signing private key
+      command: openssl genrsa -out /tmp/{{ ansible_operator_meta.name }}-work-private-key.pem 4096
+      no_log: "{{ no_log }}"
+    - name: Generate Receptor work signing public key
+      command: openssl rsa -in /tmp/{{ ansible_operator_meta.name }}-work-private-key.pem \
```

Comment:
same concerns here

Reply:
another good point

Reply:
addressed the comment by using a tempfile